test/unit/object/test_item.py

```diff
@@ -12,6 +12,30 @@ def test_item_and_response(test_file, test_folder, mock_file_response, mock_fold
     return test_folder, mock_folder_response
 
 
+@pytest.fixture(params=('empty', 'same', 'other'))
+def test_collections_for_addition(mock_collection_id, request):
+    other_collection_id = mock_collection_id + '2'
+    if request.param == 'empty':
+        return [], [{'id': mock_collection_id}]
+    elif request.param == 'same':
+        return [{'id': mock_collection_id}], [{'id': mock_collection_id}, {'id': mock_collection_id}]
+
+    return [{'id': other_collection_id}], [{'id': other_collection_id}, {'id': mock_collection_id}]
```

Comment:
My personal preference would be to have something like:

```python
elif request.param == 'other':
    return [....]

throw NotImplementedError("Forgot to implement {}".format(request.param))
```

Same for the next fixture too.